ledger/metrics/src/main/scala/com/daml/metrics/AkkaInstrumented.scala

```diff
@@ -3,11 +3,13 @@
 
 package com.daml.metrics
 
-import akka.stream.scaladsl.Source
+import akka.stream.scaladsl.{Flow, Source}
 import akka.stream.{BoundedSourceQueue, Materializer, OverflowStrategy, QueueOfferResult}
 import com.codahale.metrics.{Counter, Timer}
 
-object InstrumentedSource {
+import scala.util.chaining._
+
+object AkkaInstrumented {
```

Comment:
naming nitpick: it is slightly misleading, since akka is a different library than akka-streams. AkkaStreamsInstrumented sounds a little bit too long and also general. How about InstrumentedGraph?

Reply:
Ok 👌